src/main/java/uk/nhs/tis/trainee/notifications/model/GmcRejectedEvent.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright 2024 Crown Copyright (Health Education England)
```

Comment:
I'll just go with this was written in 2024 and only committed in 2025... so it's fine?